main.js

```diff
@@ -160,4 +160,151 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// OPDRACHT 1
+//opdracht 1a
+const televisionTypes = inventory.map((television) => {
+  return television.type;
+})
+console.log(televisionTypes);
+
+//opdracht 1b
+const soldOut = inventory.filter((television) =>{
+  if (television.originalStock - television.sold === 0){
+    return true;
+  }
+})
+console.log(soldOut);
+
+//opdracht 1c
+const containsAmbi = inventory.filter((television) =>{
+  if (television.options.ambiLight === true ){
+    return true;
+  }
+})
+console.log(containsAmbi);
+
+//opdracht 1d
+
+function sortingTv (){
+inventory.sort((a,b) => {
+  return a.price-b.price;
+})
+console.log(inventory);
+}
+// sortingTv(inventory); -->  Even uitgezet, want anders wordt Array gesorteerd
+
+
+//OPDRACHT 2
+//opdracht 2a
+let soldTelevisions=0;
+for (let i=0; i<inventory.length; i++){
+ soldTelevisions = soldTelevisions + inventory[i].sold;
```

Comment:
Je kunt hier gebruikmaken van de `+=`
```suggestion
 soldTelevisions += inventory[i].sold;
```